source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -244,54 +228,43 @@ in-memory numerical :txtref:`dataset`. It allows:
 
 For details, see :txtref:`accessors` section.
 
-Use-case example for table, accessor and table builder
-------------------------------------------------------
-
-This section provides a basic usage scenario of the :txtref:`table`,
-:txtref:`table-builder`, and :txtref:`accessor` concepts and demonstrates the
-relations between them. :txtref:`The following diagram
-<data_management_sequence_diagram>` shows objects of these concepts, which are
-highlighted by colors:
+Example of interaction between table and accessor objects
+---------------------------------------------------------
 
-- :txtref:`Table builder <table-builder>` objects are blue.
+This section provides a basic usage scenario of the :txtref:`table` and
+:txtref:`accessor` concepts and demonstrates the relations between them.
+:txtref:`The following diagram <table_accessor_usage_example>` shows objects
+of these concepts, which are highlighted by colors:
 
-- :txtref:`Table <table>` objects are cyan.
+- :txtref:`table` object is dark blue.
 
-- :txtref:`Accessors <Accessor>` are yellow.
+- :txtref:`accessor` is orange.
 
-- Grey objects are not a part of |dal_short_name| specification and they are
-  provided just for illustration purposes.
+- :txtref:`table_metadata` is light blue.
 
-.. _data_management_sequence_diagram:
+.. _table_accessor_usage_example:
 
-.. image:: _static/table_builder_accessor_sequence.png
+.. image:: _static/table_accessor_usage_example.png
```

Comment:
comments for the picture:
- remove red way underlines from the pic (right-click on the underlined phrase and click ignore)
- "Constructing **a** table"
- "some" in "some user-defined data" doesn't really add any value/info, you can easily remove it
- "Inferring **the** table metadata"
- "**An** example of direct operations"
-  "Constructing **a** read-only accessor"
- "assures" is not a good choice of wording here
- "The objects' lifetimes"

Reply:
Thanks, corrected